{{cookiecutter.package_name}}/Jenkinsfile

```diff
@@ -0,0 +1,5 @@
+@Library('vivarium_build_utils') _
+reusable_pipeline(scheduled_branches: ['main'],
+                  skip_build: true,
+                  upstream_repos: ['vivarium', 'vivarium_public_health', 'gbd_mapping', 'vivarium_cluster_tools', 'vivarium_inputs'],
```

Comment:
I'm not sure if we use LayeredConfigTree directly, but it is an upstream repo, since it's upstream of vivarium and others. I don't think the dependency checker recursively looks for dependencies, but it probably should? Can you file a ticket for that?